### PR TITLE
DPC-714 fix smoke tests

### PR DESCRIPTION
**Pull request by whuang85**
### Fixes [DPC-714](https://jira.cms.gov/browse/DPC-714)

Smoke tests weren't ran before merging in DPC-714, so this ticket is fixing the smoke test by having the test expect operation outcomes.  This will be further refined in the future when we use non mock blue button client and can skip look back.

### Proposed Changes

remove the expectation that the job has no operation outcomes

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### CODE REVIEW

File: dpc-smoketest/src/main/java/gov/cms/dpc/testing/smoketests/ClientUtils.java [hunk 1]
```diff
@@ -62,8 +62,9 @@ static void handleExportJob(IGenericClient exportClient, List<String> providerNP
                 .map(search -> (Group) search.getEntryFirstRep().getResource())
                 .map(group -> jobCompletionLambda(exportClient, httpClient, group, overrideURL))
                 .peek(jobResponse -> {
-                    if (jobResponse.getError().size() > 0)
-                        throw new IllegalStateException("Export job completed, but with errors");
+                    //TODO: ignore until we can skip lookback per request and switch over to use real bfd client
+//                    if (jobResponse.getError().size() > 0)
+//                        throw new IllegalStateException("Export job completed, but with errors");
                 })
```
Comment (jonfulk) [inline dpc-smoketest/src/main/java/gov/cms/dpc/testing/smoketests/ClientUtils.java:68]: is it possible to comment out the entire call to `.peek()`?

**Review by jonfulk — APPROVED** [review]
Minor suggestion, but this should work!

File: dpc-smoketest/src/main/java/gov/cms/dpc/testing/smoketests/ClientUtils.java (hunk 1 above)
Comment (whuang85) [reply dpc-smoketest/src/main/java/gov/cms/dpc/testing/smoketests/ClientUtils.java:68]: good point, I'll do that

**Review by jonfulk — APPROVED** [review]

**Review by dhgreene — COMMENTED** [review]
I see this working in `dev` now--great!  That will take care of our immediate needs and DPC-570 will take care of our long-term needs.  💯

**Pull request merged**